mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Membrane.H264.FFmpeg.Plugin.MixProject do
   use Mix.Project
 
-  @version "0.16.1"
+  @version "0.16.2"
```

Comment:
You can leave it here, but for future: the version bump should come in a separate PR (e.g. to prevent conflicts when there's more than one PR queued for release)

Reply:
An update here, 0.16.2 has been released so you need to change that

Reply:
Isn't it just the way it should be now? Actually I bumped the version and got a merge conflict, so if you don't mind I think I'd rather follow the usual procedure and bump it in separate PR. Sorry about the mess in commits, I didn't want to force push anything, I'll make sure to squash them before merging